tests/sles4sap/publiccloud/hana_sr_test_secondary.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Test module for performing database stop using various methods on secondary HANA database site.
+
+use base 'sles4sap_publiccloud_basetest';
+use strict;
+use warnings FATAL => 'all';
+use sles4sap_publiccloud;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    my ($self, $run_args) = @_;
+    $self->{instances} = $run_args->{instances};
+    my $hana_start_timeout = bmwqemu::scale_timeout(600);
+    # $site_b = $instance of secondary instance located in $run_args->{$instances}
+    my $site_b = $run_args->{site_b};
```

Comment:
I wonder whether we should check for `undef` here in case `$run_args->{site_b}` does not exist.